airflow/providers/amazon/aws/hooks/emr.py

```diff
@@ -315,6 +309,32 @@ def get_state(self, response, keys) -> str:
         )
         return get_state(response=response, keys=keys)
 
+    def cancel_running_jobs(self, application_id: str, waiter_config: dict = {}):
+        """
+        List all jobs in an intermediate state and cancel them.
+        Then wait for those jobs to reach a terminal state.
+
+        Note: if new jobs are triggered while this operation is ongoing,
+        it's going to time out and return an error.
+        """
+        r = self.conn.list_job_runs(
```

Comment:
I would use the paginator to get all job runs instead of only the one in first page

Reply:
ok 🥲

Reply:
This is super easy :) https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/emr-containers/paginator/ListJobRuns.html

Reply:
🤯 😍